runtime/evm_tracer/src/call_list.rs

```diff
@@ -96,6 +96,12 @@ struct Context {
 	to: H160,
 }
 
+impl Default for CallListTracer {
+	fn default() -> Self {
+		Self::new()
+	}
+}	
```

Comment:
@tgmichel @nanocryk I've added this because of this clippy lint https://rust-lang.github.io/rust-clippy/master/#new_without_default. However, I haven't used it anywhere.

1. Any opposition to me replacing calls to `new()` with calls to `default()`?
2. Any opposition to me removing the `fn new` entirely?

Reply:
@nanocryk confirmed I can remove it in DM. I've done that in c98cf71